### PR TITLE
Copy and chmod mysql configs for windows...

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: config/mysql/master/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+rm -rf /etc/mysql/conf.d/*.cnf
+cp /tmp/mwdd/master.cnf /etc/mysql/conf.d/master.cnf
```
Comment (gbirke) [inline config/mysql/master/entrypoint.sh]: I don't understand where that file is coming from ...

Comment (addshore) [reply config/mysql/master/entrypoint.sh]: /tmp/mwdd/master.cnf ?

It's coming from the mount:
"- ./config/mysql/slave:/tmp/mwdd"

File: config/mysql/master/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# On Windows if we mount the file directly it will end up having 777 permissions and mysql won't read the config
+# So instead mount to a temporary directory and copy rom there, then chmoding to 0444
```
Comment (gbirke) [inline config/mysql/master/entrypoint.sh]: small typo `rom`

**Comment by addshore** [comment]
When merged it would be great to squish / squash my changes :)

**Pull request merged**